spyder/widgets/mixins.py

```diff
@@ -755,7 +905,7 @@ def show_object_info(self, text, call=False, force=False):
                             tiptext = name + argspec
                         else:
                             tiptext = signature
-                        self.show_calltip(_("Arguments"), tiptext,
+                        self.show_tooltip(_("Arguments"), tiptext,

```

Comment:
Why this change?

Reply:
https://github.com/spyder-ide/spyder/pull/9140#discussion_r275781064

Reply:
I understand the reasoning you mentioned above, but a call to `show_object_info` (where this code is placed) shows the signature of a callable in our editor as well as its docstring rendered in our help pane.

So I think this should continue to be `show_calltip` and not `show_tooltip`.

Reply:
Fixed!